dev-libs/openssl/openssl-1.0.2f.recipe

```diff
@@ -40,6 +40,11 @@ REQUIRES="
 	lib:libz$secondaryArchSuffix >= 1.2.3
 	ca_root_certificates
 	"
+if [ -n "$secondaryArchSuffix" ]; then
+	REQUIRES="$REQUIRES
+		openssl == $portVersion base
+		"
+fi
```

Comment:
I don't care about 1.0.0 and 1.0.1 recipes. They will be dropped in a near future.